### DISCUSSION TITLE
Pro

### DISCUSSION THREAD

**Discussion by ERLANRAHMAT** [ERLANRAHMAT/BETABOTZ-MD2#36] [Q&A]
New Update Betabotz-Md2 Script

 - Use New Version Baileys
- New Ui
- New Feature

Upadate All Script

https://github.com/ERLANRAHMAT/BETABOTZ-MD2/blob/pro/README.md

<hr /><em>This discussion was created from the release <a href='https://github.com/ERLANRAHMAT/BETABOTZ-MD2/releases/tag/Pro'>Pro</a>.</em>

**Comment by Nayla-Hanifah** [comment]
ganti branch

Pada Min, 21 Jan 2024 9.15 AM, Lann ***@***.***> menulis:
